build.gradle

```diff
@@ -14,5 +14,5 @@ dependencies {
 gradlerio{
 	teamNumber '1758'
 	robotClass 'org.usfirst.frc.team1758.robot.Robot'
-	rioIP '10.17.58.22'
+	rioIP '10.17.58.52'
```

Comment:
We need to setup everyone's machine with Bonjour. That would allow us to remove this line of code and not worry about IP addresses.